packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -111,44 +133,67 @@ export default {
         getRatingStarPercentage () {
             return `${(this.starRating / this.maxStarRating) * 100}%`;
         }
+    },
+
+    methods: {
+        /**
+         * Display review rating message format.
+         *
+         * @todo - If the component is using `short` as a `starRatingSize` we shouldn't display text
+         * alongside it for now. (TBC with design - ticket in backlog).
+         *
+         * @returns {string|boolean}
+         */
+        getRatingDisplayFormat () {
+            return this.ratingDisplayType
```

Comment:
seems unnecessary as it is only being called when v-if check for `ratingDisplayType` is true?

Reply:
I see what you mean. Will fix. 👍

Reply:
Fixed.